### DISCUSSION TITLE
IP address not matching with sigma rules in playbook SO 2.3.120

### DISCUSSION THREAD

**Discussion by btdewey** [Security-Onion-Solutions/securityonion#8474] [Unsupported Versions] [answered]
Running 2.3.120 in a distributed model.  I'm having issues getting IP addresses to match on the client.ip field from zeek smb logs when using playbook.  I found a reference to a fix for this in 2.3.40 (https://github.com/Security-Onion-Solutions/securityonion/issues/3327).  There is very little data in this fix so I'm not sure what was broke and what got fixed.  If I manually test and remove the .security field it matches fine.   Should I be using a different field for matching IP addresses or is there something I can do to fix the matching in the short term?

**Comment by defensivedepth** [comment, marked as answer]
Do the zeek logs in question have `source.ip` and `destination.ip` ? Those are the IP fields that should be used.

`client.ip` does not have the needed mappings for Playbook.

**Reply by btdewey** [reply]
I'm not certain and I don't have access to that environment anymore.  I'll try to generate some logs in a test environment and see if it has the source.ip and destination.ip fields.

**Reply by defensivedepth** [reply]
It should - we copy the zeek native IP fields to source.ip & destionation.ip as part of ECS normalization.
